site/docs/4.3/getting-started/javascript.md

```diff
@@ -209,3 +209,45 @@ $('#yourTooltip').tooltip({
   }
 })
 {% endhighlight %}
+
+## Compatibility with IE 11
+
+Our v5 isn't designed to work with Internet Explorer 11, but you can add the following polyfills to make it work:
+
+{% capture callout %}
+Note that while we continue adding new features, it's possible that Internet Explorer 11 compatibility breaks in the future.
```

Comment:
We can probably skip this—we don't support IE11, so we shouldn't have to warn against it possibly breaking in the future.